README.md

```diff
@@ -65,15 +73,128 @@ href="https://github.com/GoogleCloudPlatform/terraform-example-foundation-app/tr
+    ├── prj-c-interconnect
+    ├── prj-c-logging
+    ├── prj-c-scc
+    └── prj-c-secrets
+└── fldr-development
+    ├── prj-bu1-d-boa-anthoshub
+    ├── prj-bu1-d-boa-gke
+    ├── prj-bu1-d-boa-ops
+    ├── prj-bu1-d-boa-sec
+    ├── prj-bu1-d-boa-sql
+    ├── prj-d-monitoring
+    ├── prj-d-secrets
+    ├── prj-d-shared-base
+    └── prj-d-shared-restricted
+└── fldr-non-production
+    ├── prj-bu1-n-boa-anthoshub
+    ├── prj-bu1-n-boa-gke
+    ├── prj-bu1-n-boa-ops
+    ├── prj-bu1-n-boa-sec
+    ├── prj-bu1-n-boa-sql
+    ├── prj-n-monitoring
+    ├── prj-n-secrets
+    ├── prj-n-shared-base
+    └── prj-n-shared-restricted
+└── fldr-production
+    ├── prj-bu1-p-boa-anthoshub
+    ├── prj-bu1-p-boa-gke
+    ├── prj-bu1-p-boa-ops
+    ├── prj-bu1-p-boa-sec
+    ├── prj-bu1-p-boa-sql
+    ├── prj-p-monitoring
+    ├── prj-p-secrets
+    ├── prj-p-shared-base
+    └── prj-p-shared-restricted
+```
+
+### [5. infrastructure](./5-infrastructure)
+
+The purpose of this step is to deploy the infrastructure for the Bank of Anthos example application using the infra pipeline setup in 4-projects. There is also a [Source Repository](https://cloud.google.com/source-repositories) to push the code to be deployed. All infrastructure components will be created using the base network created during step [3-networks-extension](./app-foundation/3-networks/). Curretnly, this includes:
+
+- **3 GKE Clusters**
+  - Cluster1 in the primary region (us-east1)
+  - Cluster2 in the secondary region (us-west1)
+  - MCI Cluster in the primary region (us-east1)
+- **Bastion Host VM** in the secondary region (us-west1)
+- **2 Postgres CLoudSQL instances** in the primary and secondary regions, respectively
+- **Secret** to store the CloudSQL Admin Password
+- **4 KMS Keyrings and Keys**
+  - 2 KMS Keyrings and Keys for GKE, one in each region
+  - 2 KMS Keyrings and Keys for CloudSQL, one in each region
+- **Service Account for KMS** to own/manage the Keyrings and Keys
+- **Service Account for the Bastion Host VM** with roles to install Anthos Service Mesh
+- **4 Log Sinks**, one in each project
+- **Log Sink Destination Storage Bucket** that Log Sinks write logs to
+- **Cloud Armor Policy**
+- **External IP** for accessing the application externally
+
+### [6. anthos-install](./6-anthos-install)
+
+The purpose of this step is to install the Anthos components required for the Bank of Anthos example application - Anthos Config Management and Anthos Service Mesh. **This step in the process is currently manual.**  Anthos Config Management can help you create a common configuration across all your infrastructure, including custom policies, and apply it both on-premises and across clouds. Anthos Service Mesh lets you easily manage the security and telemetry of complex environments.  Currently, this includes:
+
+- All required components to install **Anthos Config Management**
+- All required components to install **Anthos Service Mesh**
+
+### [7. build_app](./build_app)
```

Comment:
Where do we have the step by step instructions for pushing the source code to the repo to be built by this pipeline?

Reply:
the step by step for the namespace repos exists in 6-anthos-install readme, the bank-of-anthos-source repo step by step will be in the build_app folder readme though currently atm it does not have them.

Reply:
sg, lets open an issue to track the missing readme for that